### PR TITLE
Create code file for CPX Simon Guide

### PR DESCRIPTION
**Pull request by TheKitty**

### CODE REVIEW

File: CPX_Simon_Game/code.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import time
+import random
+from adafruit_circuitplayground.express import cpx
+
+cpx.pixels.brightness = 0.1  # adjust NeoPixel brightness to your liking
+
+REGION_LEDS = [
```
Comment (ladyada) [inline CPX_Simon_Game/code.py]: things that dont change should be tuples () not lists []

File: CPX_Simon_Game/code.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+]
+
+REGION_COLOR = [
+    (255, 255, 0),  # yellow region
+    (0, 0, 255),    # blue region
+    (255, 0, 0),    # red region
+    (0, 255, 0),    # green region
+]
+
+REGION_TONE = [
```
Comment (ladyada) [inline CPX_Simon_Game/code.py]: ditto

File: CPX_Simon_Game/code.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+cpx.pixels.brightness = 0.1  # adjust NeoPixel brightness to your liking
+
+REGION_LEDS = [
+    [5, 6, 7],  # yellow region
+    [2, 3, 4],  # blue region
+    [7, 8, 9],  # red region
+    [0, 1, 2],  # green region
+]
+
+REGION_COLOR = [
```
Comment (ladyada) [inline CPX_Simon_Game/code.py]: ditto

Comment (TheKitty) [reply CPX_Simon_Game/code.py]: Limor, the code works and the guide is ready. Would you like me to have the author make the tuple changes? He'll probably have to do it after work and then test it.

Comment (TheKitty) [reply CPX_Simon_Game/code.py]: @ladyada - code now lints, shall I ask the author to make the tuple changes?

Comment (ladyada) [reply CPX_Simon_Game/code.py]: please do it for them and let them know you did it

Comment (TheKitty) [reply CPX_Simon_Game/code.py]: Right-o

**Pull request merged**